config/default.py

```diff
@@ -32,11 +32,13 @@ class Default:
 
     # pylint: disable=invalid-name
     PROJECT_ID: str = os.environ.get("PROJECT_ID")
+    DEMO_PROJECT_ID: str = os.environ.get("DEMO_PROJECT_ID")
```

Comment:
What's the difference between PROJECT_ID and DEMO_PROJECT_ID ?

Reply:
Good point, both can be the same. The DEMO_PROJECT_ID here is defaulting to n25 but the PROJECT_ID is mine. Let me just revise this PR and standardize it. It shouldn't have to matter.